### PR TITLE
[MOB-XXXX] updates changelog

### PR DESCRIPTION
**Pull request by evantk91**
## 🔹 Jira Ticket(s) if any

* [MOB-XXXX](https://iterable.atlassian.net/browse/MOB-XXXX)

## ✏️ Description

> This pull request updates the change log for embedded GA release.

### CODE REVIEW

File: CHANGELOG.md
Comment (Ayyanchira) [inline CHANGELOG.md:1]: We have kept the unreleased portion on top as template to copy paste quickly.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,15 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [3.5.0]
```
Comment (Ayyanchira) [inline CHANGELOG.md:5]: Do you want to merge this now? Or make it a separate PR where library versions also change with this?

File: CHANGELOG.md
Comment (evantk91) [reply CHANGELOG.md:1]: Oh gotcha

**Review by Ayyanchira — APPROVED** [review]
LGTM
